src/components/common/ProductBox/ProductBox.js

```diff
@@ -16,6 +16,22 @@ const addFurnitureToFavourite = (id, isFavourite, event, addFavourite) => {
   addFavourite({ id: id, isFavourite: !isFavourite });
 };
 
+const toggleCompare = (
+  id,
+  compare,
```

Comment:
Dodajmy tutaj może `is` żeby było `isComapre` będzie czytelniej

Reply:
Zrobione